modules/thunder_paragraphs/thunder_paragraphs.install

```diff
@@ -101,3 +101,29 @@ function thunder_paragraphs_update_8001() {
 
   return $updateLogger->output();
 }
+
+/**
+ * Change media paragraph to use inline_entity_form.
+ */
+function thunder_paragraphs_update_8101() {
+  /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
+  $thunderUpdater = \Drupal::service('thunder_updater');
+  $updateLogger = $thunderUpdater->logger();
+
+  $config = \Drupal::configFactory()->getEditable('core.entity_form_display.paragraph.media.default');
```

Comment:
It would be best to make 3 cases here:
1. we should check if media paragraph exists on system at all and if not -> we check `v1_1__media_paragraph` in CheckList and log that nothing is changed and everything is good (so `info()` log)
2. if media paragraph exists + we execute update successfully -> we mark update as successful, but with unchecked `v1_1__media_paragraph`, so that we ensure admin will ready notification of deprecation
3. if media paragraph exists + update fails, because field widget is not "rendered_entity" -> then we mark update as failed